src/platform/macos/window.rs

```diff
@@ -94,22 +96,30 @@ impl DelegateState {
         }
     }
 
+    unsafe fn saved_style_mask(&self, resizable: bool) -> NSWindowStyleMask {
+        let base_mask = self.save_style_mask
+            .take()
+            .unwrap_or_else(|| self.window.styleMask());
+        if resizable {
+            base_mask | NSWindowStyleMask::NSResizableWindowMask
+        } else {
+            base_mask & !NSWindowStyleMask::NSResizableWindowMask
+        }
+    }
+
+    fn saved_standard_frame(&self) -> NSRect {
+        self.standard_frame.get().unwrap_or(NSRect::new(
```

Comment:
```suggestion
        self.standard_frame.get().unwrap_or_else(|| NSRect::new(
```
`unwrap_or` will unconditionally construct `NSRect`: https://doc.rust-lang.org/core/option/enum.Option.html#method.unwrap_or